dev-util/scons/files/scons-3.0.5-jdk-include-path.patch

```diff
@@ -0,0 +1,15 @@
+diff -Nur old/scons-3.0.5/src/engine/SCons/Tool/JavaCommon.py new/scons-3.0.5/src/engine/SCons/Tool/JavaCommon.py
+--- old/src/engine/SCons/Tool/JavaCommon.py	2019-03-27 02:16:32.000000000 +0300
++++ new/src/engine/SCons/Tool/JavaCommon.py	2019-06-04 10:44:01.000000000 +0300
+@@ -403,7 +403,10 @@
+ java_macos_version_include_dir = '/System/Library/Frameworks/JavaVM.framework/Versions/%s*/Headers/'
+ 
+ java_linux_include_dirs = ['/usr/lib/jvm/default-java/include',
+-                        '/usr/lib/jvm/java-*/include']
++                        '/usr/lib/jvm/java-*/include',
++                        '/opt/oracle-jdk-bin-*/include',
```

Comment:
Wouldn't `/usr/lib/jvm/*/include` be easier here? I suppose if you replace the `java-*` above, you can even submit it upstream.

Reply:
I'll try to modify it this way and check.

The upstream changed the logic of JavaCommon in this commit: https://github.com/SCons/scons/commit/b774c488
but it's reasonable to submit similar patch to upstream.